### PR TITLE
[IMP]10705 add module mrp_production_xls_report

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: mrp_production_xls_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Mrp Production Report Xls
```
Comment (elicoidal) [inline mrp_production_xls_report/README.rst]: MRP

File: mrp_production_xls_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+=========================
+
+This module extends the functionality of MEP to provide Mrp Production report in landscape mode.
```
Comment (elicoidal) [inline mrp_production_xls_report/README.rst]: This module extends the functionality of MRP and provides XLS Production report in landscape mode.

File: mrp_production_xls_report/README.rst (hunk 1 above)
Comment (elicoidal) [inline mrp_production_xls_report/README.rst]: XLS

File: mrp_production_xls_report/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Mrp Production Xls Report',
+    'version': '8.0.1.0.1',
+    'category': 'hr',
```
Comment (elicoidal) [inline mrp_production_xls_report/__openerp__.py]: Manufacturing

**Comment by elicoidal** [comment]
There is a lot of Chinese/farsi in the code that should be cleansed before publishing
Farsi should be add in the farsi translation

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Pull request merged**